src/core/manager.c

```diff
@@ -1051,7 +1051,7 @@ static int manager_setup_user_lookup_fd(Manager *m) {
                  * resolutions */
                 r = sd_event_source_set_priority(m->user_lookup_event_source, SD_EVENT_PRIORITY_NORMAL-11);
                 if (r < 0)
-                        return log_error_errno(errno, "Failed to set priority ot user lookup event source: %m");
+                        return log_error_errno(errno, "Failed to set priority to user lookup event source: %m");
```

Comment:
All of the other cases in here say `Failed to set priority of …` which I guess was the original typo anyways rather than a transposition. Should I keep uniformity with `of` here?

Reply:
Yeah, "of" is OK.